presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -384,15 +384,15 @@ public void checkCanCreateSchema(SystemSecurityContext context, CatalogSchemaNam
     public void checkCanDropSchema(SystemSecurityContext context, CatalogSchemaName schema)
     {
         if (!isSchemaOwner(context, schema)) {
-            denyDropSchema(schema.getSchemaName());
+            denyDropSchema(schema.toString());
```

Comment:
why this is better? I would undo this change. It is more clear to use domain objects like schema name here instead of `toString()` representation (which is not intuitive that it is designed for UX message)

Reply:
The default method in the super class do this, so this is consistent with other system level items.  I actually would like the connector access control to give absolute names, but they don't really have that information easily.

BTW, I would guess these are written this way because this code was copied from the connector level file access control.